### ISSUE TITLE
Kdenlive should be have frei0r as a build input

### ISSUE THREAD

**Issue by lambda-11235** [NixOS/nixpkgs#66157]
**Describe the bug**
Without frei0r none of the effects operate. Installing frei0r separately does not work.

**To Reproduce**
Try to use an effect in Kdenlive. You'll get an inoperable UI element.

**Expected behavior**
The appropriate UI element should appear.

**Additional context**
Fixed by changing buildInputs to
```
  buildInputs = [
    breeze-gtk
    breeze-icons
    breeze-qt5
    frei0r
  
    kconfig
    kcrash
    kdbusaddons
    kfilemetadata
    kguiaddons
    ki18n
    kiconthemes
    kinit
    knotifications
    knewstuff
    karchive
    knotifyconfig
    kplotting
    ktextwidgets
    mlt
    phonon-backend-gstreamer
    qtdeclarative
    qtquickcontrols
    qtscript
    qtwebkit
    shared-mime-info
    libv4l
    ffmpeg
    rttr
    kpurpose
    kdeclarative
  ];
```

**Metadata**
Please run `nix run nixpkgs.nix-info -c nix-info -m` and paste the result.
 - system: `"x86_64-linux"`
 - host os: `Linux 5.1.21, NixOS, 19.03.173238.4b6dd53b90a (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2.2`
 - channels(root): `"nixos-19.03.173238.4b6dd53b90a"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: kdenlive
# a list of nixos modules affected by the problem
module:
```

**Comment by worldofpeace** [comment]
Sounds correct to me @lambda-11235, care to PR?

**Comment by lambda-11235** [comment]
Fixed with merge of PR #66220.
